Makefile

```diff
@@ -95,13 +96,32 @@ help: ## Print help for targets with comments.
 # Without the check there is a race condition between the link being deleted
 # and recreated which can break the addons build when running test-ci
 # See comments on the build-addons target for some more info
+ifeq ($(BUILD_WITH), make)
 $(NODE_EXE): config.gypi out/Makefile
 	$(MAKE) -C out BUILDTYPE=Release V=$(V)
 	if [ ! -r $@ -o ! -L $@ ]; then ln -fs out/Release/$(NODE_EXE) $@; fi
 
 $(NODE_G_EXE): config.gypi out/Makefile
 	$(MAKE) -C out BUILDTYPE=Debug V=$(V)
 	if [ ! -r $@ -o ! -L $@ ]; then ln -fs out/Debug/$(NODE_EXE) $@; fi
+else
+ifeq ($(BUILD_WITH), ninja)
+ifeq ($(V),1)
+  NINJA_ARGS := $(NINJA_ARGS) -v
```

Comment:
Perhaps the `-jN` flag, if it's present in `MAKEFLAGS`, should be copied into `NINJA_ARGS`? So that `make -j2` ends up calling `ninja -j2`.

Reply:
> Perhaps the `-jN` flag, if it's present in `MAKEFLAGS`, should be copied into `NINJA_ARGS`? So that `make -j2` ends up calling `ninja -j2`.

Added support via a `JOBS` env var.

FTR: By [default `ninja` runs in multiproc mode](https://ninja-build.org/manual.html#_running_ninja) so adding `-j` is used to change the default or limit the number of parallel processes.
Also `make` or even `gmake` do [not readily expose](https://blog.jgc.org/2015/03/gnu-make-insanity-finding-value-of-j.html) the value that was passed to them.

Reply:
Added `$(filter -j%,$(MAKEFLAGS))`